backend/src/index.js

```diff
@@ -22,8 +22,8 @@ app.get('/', (req, res) => {
   res.send('hi')
 })
 
-app.use('/candidates', candidates)
-app.use('/matchCandidates', matchCandidates)
+app.use('/', candidates)
+app.use('/', matchCandidates)
```

Comment:
@tko22 This feels a little bad but since the endpoints in each file don't share a common prefix anymore, I thought this was the best way. The other alternative I can think of is renaming the paths of related endpoints to share a common prefix.

Reply:
ah sorry for the confusion. I didn't mean changing it in `index.js`. just renaming the router to be `router = express.Router()` in both `candidates.js` and `matchCandidates.js` instead of it being `candidates = express.Router()`. I agree with you that it should be 
```Javascript
app.use('/candidates', candidates)
app.use('/matchCandidates', matchCandidates)
```